ampd/src/handlers/multisig.rs

```diff
@@ -138,8 +150,8 @@ where
         match pub_keys.get(&self.verifier) {
             Some(pub_key) => {
                 let key_type = match pub_key {
-                    PublicKey::Secp256k1(_) => tofnd::Algorithm::Ecdsa,
-                    PublicKey::Ed25519(_) => tofnd::Algorithm::Ed25519,
+                    PublicKey::Secp256k1(_) => tofnd::Algorithm::Ed25519,
+                    PublicKey::Ed25519(_) => tofnd::Algorithm::Ecdsa,
```

Comment:
this is not right, this is what bricked ampd signing sometime ago. these lines shouldn't be changed

Reply:
Fixed: https://github.com/axelarnetwork/axelar-amplifier/pull/764/commits/a10002baa461518dc073732e26291dec17f0f42a